src/radical/pilot/utils/prof_utils.py

```diff
@@ -199,6 +199,114 @@
+        'u_agent_execute_queue'        : [ {ru.EVENT: 'state'           , ru.STATE: rps.AGENT_EXECUTING_PENDING     },  
+                                           {ru.EVENT: 'state'           , ru.STATE: rps.AGENT_EXECUTING             }  ], 
+        'u_agent_execute_prepare'      : [ {ru.EVENT: 'state'           , ru.STATE: rps.AGENT_EXECUTING             },  
+                                           {ru.EVENT: 'exec_mkdir'      , ru.STATE: None                            }  ], 
+        'u_agent_execute_mkdir'        : [ {ru.EVENT: 'exec_mkdir'      , ru.STATE: None                            },
+                                           {ru.EVENT: 'exec_mkdir_done' , ru.STATE: None                            }  ],
+        'u_agent_execute_layer_start'  : [ {ru.EVENT: 'exec_mkdir_done' , ru.STATE: None                            },
+                                           {ru.EVENT: 'exec_start'      , ru.STATE: None                            }  ],
+        'u_agent_execute_layer'        : [ {ru.EVENT: 'exec_start'      , ru.STATE: None                            },
+                                          [{ru.EVENT: 'exec_ok'         , ru.STATE: None                            },
+                                           {ru.EVENT: 'exec_fail'       , ru.STATE: None                            } ]], # orte, ssh, mpi, ... 
+        'u_agent_cu_start'             : [ {ru.EVENT: 'cu_start'        , ru.STATE: None                            },
+                                           {ru.EVENT: 'cu_cd_done'      , ru.STATE: None                            }  ], # PROBLEM: discontinuity
+        'u_agent_cu_pre_execute_start' : [ {ru.EVENT: 'cu_cd_done'      , ru.STATE: None                            },
+                                           {ru.EVENT: 'cu_pre_start'    , ru.STATE: None                            }  ],
+        'u_agent_cu_pre_execute'       : [ {ru.EVENT: 'cu_pre_start'    , ru.STATE: None                            },
+                                           {ru.EVENT: 'cu_pre_stop'     , ru.STATE: None                            }  ],
+        'u_agent_cu_execute_start'     : [ {ru.EVENT: 'cu_pre_stop'     , ru.STATE: None                            },
+                                           {ru.EVENT: 'cu_exec_start'   , ru.STATE: None                            }  ],
+        'u_agent_cu_execute'           : [ {ru.EVENT: 'cu_exec_start'   , ru.STATE: None                            },
+                                           {ru.EVENT: 'cu_exec_stop'    , ru.STATE: None                            }  ],
+        'u_agent_cu_stop'              : [ {ru.EVENT: 'cu_exec_stop'    , ru.STATE: None                            },
+                                           {ru.EVENT: 'cu_stop'         , ru.STATE: None                            }  ],
+        'u_agent_cu_unschedule_start'  : [ {ru.EVENT: 'exec_stop'       , ru.STATE: None                            },
```

Comment:
The durations from here on should not have the `_cu_` midfix, they are not related to the unit LM script.

Reply:
Done.